js-packages/tsconfig/README.md

```diff
@@ -22,7 +22,7 @@ A baseline `tsconfig.json` is provided below that you can modify as needed. This
   // This will match all .ts, .tsx, .d.ts, .js, .jsx files in your `src` folder
   // and also tells your Typescript server to read core's global typings for
   // access to `dayjs` and `$` in the global namespace.
-  "include": ["src/**/*", "../vendor/flarum/core/js/dist-typings/@types/**/*"],
+  "include": ["src/**/*", "../vendor/flarum/framework/js/dist-typings/@types/**/*"],
```

Comment:
I think this should remain as `core`, since this is referring to the PHP package name which is still `flarum/core`.